buildroot/share/PlatformIO/boards/BigTree_Btt002.json

```diff
@@ -21,34 +21,17 @@
   "debug": {
     "jlink_device": "STM32F407VE",
     "openocd_target": "stm32f4x",
-    "svd_path": "STM32F40x.svd",
-    "tools": {
-      "stlink": {
-        "server": {
-          "arguments": [
-            "-f",
-            "scripts/interface/stlink.cfg",
-            "-c",
-            "transport select hla_swd",
-            "-f",
-            "scripts/target/stm32f4x.cfg",
-            "-c",
-            "reset_config none"
-          ],
-          "executable": "bin/openocd",
-          "package": "tool-openocd"
-        }
-      }
-    }
+    "svd_path": "STM32F40x.svd"
   },
   "frameworks": [
     "arduino",
+    "cmsis",
     "stm32cube"
   ],
-  "name": "STM32F407VE (64k RAM. 512k Flash)",
+  "name": "STM32F407VE (192k RAM. 512k Flash)",
   "upload": {
     "disable_flushing": false,
-    "maximum_ram_size": 65536,
+    "maximum_ram_size": 196608,
```

Comment:
hmm, beware about that, they reverted that in 5.6.0

Reply:
basically, its 2 zones of RAM, which cant be merged in one malloc...

Reply:
Hmm, I see. But boards that use the same chip (e.g. [this one](https://github.com/platformio/platform-ststm32/blob/develop/boards/armed_v1.json)) have the same value here. They use a different linker script.

Reply:
https://github.com/platformio/platform-ststm32/commit/ac83bd4eb74735c8655c3a1b0fca4100ac9acc05

Reply:
seems 131072 the good value for the 407VE

Reply:
to set for the other one too so...

Reply:
its basically 128k user + 64k of Cortex M4 internal ram

Reply:
sram + ram maybe, i didnt dig further ;)

Reply:
in general, it wont be an issue because marlin doesnt require so much ram.. but well, i tilted on the 192K

Reply:
Changed it to 131072.